src/domains/sign/apim.tf

```
@@ -91,6 +101,15 @@ module "apim_io_sign_product" {
   policy_xml = file("./api_product/sign/_base_policy.xml")
 }
 
+resource "azurerm_api_management_api_operation_policy" "get_signer_by_fiscal_code_policy" {
+  api_name            = "io-sign-api"
```

```suggestion
  api_name            = module.apim_io_sign_issuer_api_v1.name
```

Two birds with one stone:
- we make sure to set the correct API name
- we define a dipendency between the two resoruces

---

Done!